Analytics/Classes/Crypto/SEGAES256Crypto.h

```diff
@@ -0,0 +1,23 @@
+//
+//  SEGAES256Crypto.h
+//  Analytics
+//
+//  Copyright © 2016 Segment. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "SEGCrypto.h"
+
+@interface SEGAES256Crypto : NSObject <SEGCrypto>
+
+@property (nonatomic, readonly, nonnull) NSString *password;
+@property (nonatomic, readonly, nonnull) NSData *salt;
+@property (nonatomic, readonly, nonnull) NSData *iv;
+
+- (instancetype _Nonnull)initWithPassword:(NSString * _Nonnull)password salt:(NSData * _Nonnull)salt iv:(NSData * _Nonnull)iv;
+// Convenient shorthand. Will randomly generate salt and iv.
+- (instancetype _Nonnull)initWithPassword:(NSString * _Nonnull)password;
+
++ (NSData * _Nonnull)randomDataOfLength:(size_t)length;
```

Comment:
Yes, it's a convenience method for people to generate `salt` and `iv` and possibly `password`.